tests/testthat/test-vectordata.R

```diff
@@ -50,8 +50,8 @@ for (func in charting.funcs)
                 # Run command to create the widget and compare expectation
                 if (grepl("labeledscatter-.*(datalabel|fit)", filestem))
                     expect_error(suppressWarnings(eval(parse(text=cmd))), NA)
-                else if (grepl("area-single", filestem))
-                    expect_error(eval(parse(text=cmd)))
+                #else if (grepl("area-single", filestem))
+                #    expect_error(eval(parse(text=cmd)))
```

Comment:
These were commented out. Should these comments be be removed entirely? It seems that it will enter the final else clause which evaluates these area-single cases without error or warning now when it was expected to error previously?

Reply:
Thanks, I forgot to remove these when I finished testing. That's right, Area charts no longer give an error in the area-single cases and enter the final clause.